### DISCUSSION TITLE
how to install czkawka in macOS

### DISCUSSION THREAD

**Discussion by Shehrozie** [qarmin/czkawka#1214] [General]
Hi, Can anybody tell me how to i install czkawka in macOS

**Comment by qarmin** [comment]
Binaries for latest version have been deployed yesterday - https://github.com/qarmin/czkawka/releases/tag/7.0.0, so probably just downloading and running them should be enough

If this not works, compilation is required and compilation steps are different for each module(in each module, in Readme.md should be instruction how to compile app on each platform)

**Comment by edlime** [comment]
Forgive my ignorance but I'm getting errors on Rust and can't find any viable solution to fix it online.

```shell
❯ cargo run --all-features --release --bin czkawka_gui --verbose
`error: process didn't exit successfully: `rustc -vV` (signal: 6, SIGABRT: process abort signal)
--- stderr
dyld[29422]: Symbol not found: __ZN4llvm10PGOOptionsC1ENSt3__112basic_string.....trINS_3vfs10FileSystemEEENS0_9PGOActionENS0_11CSPGOActionEbb
  Referenced from: <669C5C....4154B7417B> /opt/homebrew/Cellar/rust/1.77.2/lib/librustc_driver-3dc17bded25d4e59.dylib
  Expected in:     <B0C4597.....ACE34F6C> /opt/homebrew/Cellar/llvm/18.1.4/lib/libLLVM.dylib`
```

Running in an M1 Apple Silicon.
